zeppelin-web/src/components/websocketEvents/websocketMsg.service.js

```diff
@@ -151,6 +151,15 @@ angular.module('zeppelinWebApp').service('websocketMsgSrv', function($rootScope,
       });
     },
 
+    importNoteFromIpfs: function(hashUrl) {
```

Comment:
I think this might be called more generic, somethin like `importNoteFromBackend` as it may be another NotebookRepo implementation configured that actuallys upports same backend-style import.
How do you think?